dbt/adapters/spark_livy/livysession.py

```diff
@@ -104,9 +109,10 @@ def create_session(self, data):
                     )
                 return
 
+            logger.debug(f"Sleeping {DEFAULT_POLL_WAIT} seconds before next poll")
```

Comment:
same here.rinse and repeat.

Reply:
you probably need to refresh, this is already fixed.